src/patternfly/layouts/Level/level.hbs

```diff
@@ -1,3 +1,6 @@
-<div class="pf-l-level{{#if modifierClass }} {{ modifierClass }}{{/if}} Documentation-container-tall">
+<div class="pf-l-level{{#if level--modifier}} {{level--modifier}}{{/if}} Documentation-container-tall"
```

Comment:
I think there is a new way to add heights to the preview, can we remove this and use the new method. If you can't find it let me know.